packages/mc-scripts/src/utils/graphql-requests.ts

```diff
@@ -112,18 +115,26 @@ async function requestWithTokenRetry<Data, QueryVariables extends Variables>(
   requestOptions: {
     variables?: QueryVariables;
     mcApiUrl: string;
-    headers: HeadersInit;
+    headers: Record<string, string>;
   },
   retryCount: number = 0
 ): Promise<Data> {
+  const shouldUseExperimentalIdentityAuthFlow =
+    process.env.ENABLE_EXPERIMENTAL_IDENTITY_AUTH_FLOW === 'true';
+
   const token = credentialsStorage.getToken(requestOptions.mcApiUrl);
 
+  const tokenHeader: Record<string, string | null> =
+    shouldUseExperimentalIdentityAuthFlow
+      ? { [SUPPORTED_HEADERS.AUTHORIZATION]: `Bearer ${token}` }
+      : { 'x-mc-cli-access-token': token };
```

Comment:
The session token now is treated as a "normal token" (like when developing MC apps locally), so we can send it via `authorization` header.

The special header `x-mc-cli-access-token` won't be needed anymore. This will simplify some parts of the MC API around that.
For the time being though, the logic will stay for backwards compatibility.